### PR TITLE
XMPP integration

### PR DESCRIPTION
**Pull request by dallerk**
This commit allows sending messages via XMPP.

### CODE REVIEW

**Review by sibalzer — COMMENTED** [review]
I will test this manually tonight, but it looks good.

File: config.ini.example [hunk 1]
```diff
@@ -24,6 +24,18 @@ token=TOKEN
 ; User-ID des Empfängers: https://t.me/userinfobot
 chat_id=123456789,987654321
 
+[XMPP]
+; true wenn eine Benachrichtigung via XMPP gesendet werden soll sonst false
+enable_xmpp=true
```
Comment (paulypeter) [inline config.ini.example:29]: should be just "enable" after refactoring

**Comment by dallerk** [comment]
I just noticed that Apprise was integrated by ToHindahl a few days ago. XMPP seems to be covered by that change as well. Should xmpppy still be integrated as well or is it obsolete?

**Comment by sibalzer** [comment]
hmm you're right, it might be better. unfortunately @ToHindahl closed the pr too early so i didn't have time to merge. if nothing happens by tomorrow i will add apprise from github history

**Pull request closed without merging**